Gems/GradientSignal/Code/Include/GradientSignal/GradientSampler.h

```diff
@@ -84,6 +89,22 @@ namespace GradientSignal
         }
     }
 
+    inline AZ::Matrix3x4 GradientSampler::GetTransformMatrix() const
```

Comment:
Mentioned in direct communication but adding here for history as well - could be a useful optimization to cache this as a member variable and then track when dirty to recompute.